evm/src/executor/fork/cache.rs

```diff
@@ -204,10 +204,11 @@ impl MemDb {
                 accounts.insert(add, acc.info);
 
                 let acc_storage = storage.entry(add).or_default();
-                if acc.filth.abandon_old_storage() {
+                if acc.storage_cleared {
```

Comment:
this was the only thing I wasn't sure about whether this identical

the previous was:

```rust
impl Filth {
    pub fn abandon_old_storage(&self) -> bool {
        matches!(self, Self::Destroyed | Self::NewlyCreated)
    }
```

wdyt @rakita

Reply:
It seems correct: https://github.com/bluealloy/revm/blob/24622a4c9e450217f6a723008a75e3faa3b7cc5d/crates/revm/src/db/in_memory_db.rs#L126